Assets/Resources/center.prefab

```diff
@@ -75,7 +76,7 @@ SpriteRenderer:
   m_PrefabParentObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 1170983175562992}
-  m_Enabled: 1
+  m_Enabled: 0
```

Comment:
I hid the center, its not good having it displayed, causes too much distraction for me.